test/index.test.js

```diff
@@ -88,16 +89,45 @@ describe('Out of Band cache', () => {
       assume(calledRefresh).is.truthy();
     });
 
+    it('can override the cache-level maxAge', async function () {
+      const fridge = new Cache({ maxAge: -10, maxStaleness: -10 }); // items are immediately stale
+      await fridge.get('good milk', { maxAge: 1000 }, () => Promise.resolve('milk'));
```

Comment:
You can just pass an `async` predicate rather than manually resolving a Promise.

```js
fridge.get('good milk', { maxAge: 1000 }, async () => 'milk');
```